test/test_examples_2d_mhdmulti.jl

```diff
@@ -12,8 +12,8 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
 
   @testset "elixir_mhdmulti_ec.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhdmulti_ec.jl"),
-      l2   = [4.30905893e-02, 4.30736556e-02, 2.41235681e-02, 1.16476161e-01, 1.68963068e-02, 1.68956288e-02, 2.61726930e-02, 3.12424788e-03, 1.08100979e-02, 2.16201959e-02],
-      linf = [3.20086766e-01, 3.10136277e-01, 1.74235540e-01, 6.52946929e-01, 1.03184941e-01, 1.02380573e-01, 1.37658856e-01, 6.05526386e-02, 7.13763330e-02, 1.42752666e-01])
+      l2   = [4.32946856e-02, 4.32790096e-02, 2.58822718e-02, 1.62024804e-01, 1.77548775e-02, 1.77564324e-02, 2.69452591e-02, 3.45826315e-03, 1.20451485e-02, 2.40902969e-02],
+      linf = [3.28937103e-01, 3.31201677e-01, 1.86307657e-01, 1.03382043e+00, 8.77373400e-02, 8.85611301e-02, 1.46918707e-01, 4.84055914e-02, 8.80607765e-02, 1.76121553e-01])
```

Comment:
Are these changes necessary because of your change of `gammas` and `gas_constants` in `examples/2d/elixir_mhdmulti_ec.jl` or were they caused by differences coming from the change of LLF flu implementation? If the latter is the case, I would be worried. If the former is the case, this looks good to me.

Reply:
I agree. If this is resolved, one of us can merge this.

Reply:
No worries, it's just because of the change of gammas! :)